crd/fleet/fleet-crds.yaml

```diff
@@ -3892,15 +5239,251 @@ spec:
+                  required:
+                  - code
+                  - createTime
+                  - resource
+                  type: object
+                type: array
+              observedGeneration:
+                description: 'Internal: The generation observed by the controller.'
+                format: int64
+                type: integer
+              reconciled:
+                description: 'Internal: Whether the resource was reconciled by the
+                  controller.'
+                type: boolean
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources: {}
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  labels:
+    alloydb-omni: "true"
+  name: failovers.alloydbomni.dbadmin.goog
+spec:
+  group: alloydbomni.dbadmin.goog
+  names:
+    kind: Failover
+    listKind: FailoverList
+    plural: failovers
+    singular: failover
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.state
+      name: state
+      type: string
+    - jsonPath: .status.internal.phase
+      name: phase
+      type: string
+    name: v1
+    schema:
+      openAPIV3Schema:
+        description: Failover represents the parameters and status of a single failover
+          operation.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: FailoverSpec represents the parameters of a single failover
+              operation.
+            properties:
+              dbclusterRef:
+                description: DBClusterRef is the DBCluster name to initiate a failover.
+                  The `Failover` object must be created in the same namespace as the
+                  DBCluster that it references. This field is required for Failover.
+                type: string
+              newPrimary:
+                description: NewPrimary is the standby instance to promote as the
+                  new primary. If left empty, the system will automatically pick the
+                  best one to failover to.
```

Comment:
What does "one" mean here? Instance? Replace "one" with the name of the thing itself to ensure user comprehension.